liblangutil/Scanner.h

```diff
@@ -68,6 +68,12 @@ class AstRawString;
 class AstValueFactory;
 class ParserRecorder;
 
+enum class ScannerKind
```

Comment:
Should this be `ScannerMode` if we use `setScannerMode`?

Reply:
I think both are fine